crates/polars-io/src/json/mod.rs

```diff
@@ -314,14 +316,16 @@ where
         self
     }
 
-    /// Set the JSON reader to infer the schema of the file. Currently, this is only used when reading from
-    /// [`JsonFormat::JsonLines`], as [`JsonFormat::Json`] reads in the entire array anyway.
+    /// Set the number of records for the JSON reader to infer the schema of the file
+    /// from.
```

Comment:
In your edit the "Currently, this is only used when reading from [`JsonFormat::JsonLines`], as [`JsonFormat::Json`] reads in the entire array anyway." got lost. Can you put that back?

Reply:
It changed — both formats now use `infer_schema_len`. `JsonFormat::Json` will read in the entire array, but can inspect a subset of records for schema inference.

Reply:
Hi, just following up on this. The removal of that part of the comment was intentional because the removed content was outdated. Now, both `JsonFormat::JsonLines` and `JsonFormat::Json` can use `infer_schema_len` to check only a subset of records when inferring the schema. (Not sure when it changed.)